docs/credentials/remove-credential.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Remove a Credential
+intro: Learn how to remove a credential when it's no longer needed
+links:
+    overview:
+    quickstart:
+    previous:
+    next:
+    guides:
+    related:
+        - /docs/credentials/add-credential
+    featured:
+---
+
+{% note %}
+
+**NOTE**: To remove a credential, it cannot be used by any environment resource. Before removing a credential, you should edit the resources that use the credential you want to remove to use another credential.
+
+{% endnote %}
+
+{% note %}
+
+**NOTE**: In case you need to rotate the credential keys, you can edit the credential. Follow the guide {% mentionPost "/docs/credentials/edit-credential" %}
```

Comment:
1. Credentials that were used in the past, but are no longer being used, can be deleted with no issues
2. I believe we should first present the option of not removing a credential (by editing/rotating its keys and secrets) and only after the reader is convinced that editing is not an option, then we can go into details of what to do before deleting a credential

```suggestion
**NOTE**: In case you need to rotate credential keys, you can edit a credential instead of removing a credential and create a new one. To edit a credential, follow the guide {% mentionPost "/docs/credentials/edit-credential" %}

**WARNING**: If you are really sure you want to remove a credential, that cannot be done if the credential to be removed is currently in use by any environment resource.
To remove a credential that is currently in use you should edit each resource using that credential to use another credential.

{% endnote %}

{% note %}
```

Reply:
@jfoliveira I believe we can use separate callbacks for note and warning messages, an example currently available of the syntax is the post below https://github.com/devopness/devopness/blob/4ef8bc89313ea70016060b0396b92e2cd05715c4/docs/servers/stop-server.md?plain=1#L20-L24

```suggestion
**NOTE**: In case you need to rotate credential keys, you can edit a credential instead of removing a credential and create a new one. To edit a credential, follow the guide {% mentionPost "/docs/credentials/edit-credential" %}

{% endnote %}

{% warning %} 

**WARNING**: If you are really sure you want to remove a credential, that cannot be done if the credential to be removed is currently in use by any environment resource.
To remove a credential that is currently in use you should edit each resource using that credential to use another credential.

{% endwarning %}
```

Reply:
@thlmenezes nice one!
@souz4-my could you please review/apply these suggestions?